### PR TITLE
OSDOCS-3568: Adding RN about future Kubernetes API removals

### PR DESCRIPTION
**Pull request by bergerhoffer**
Version(s):
4.11

Issue:
https://issues.redhat.com/browse/OSDOCS-3572

Link to docs preview:
http://file.rdu.redhat.com/~ahoffer/2022/OSDOCS-3568/release_notes/ocp-4-11-release-notes.html#ocp-4-11-future-removals

### CODE REVIEW

File: release_notes/ocp-4-11-release-notes.adoc [hunk 1]
```diff
@@ -290,6 +290,16 @@ In the table, features are marked with the following statuses:
...
 
+[id="ocp-4-11-future-removals"]
+=== Future Kubernetes API removals
+
+The next minor release of {product-title} is expected to use Kubernetes 1.25. Currently, Kubernetes 1.25 plans to remove several deprecated `v1beta1` and `v2beta1` APIs.
+
+See the link:https://kubernetes.io/docs/reference/using-api/deprecation-guide/#v1-25[Deprecated API Migration Guide] in the Kubernetes documentation for the list of planned Kubernetes API removals.
+
+// TODO: Add a call to action for users now to stop using these APIs?
+// TODO: Do we want to point to the KCS article now? Can admins actually provide the ack now, or will they have to do that right before they actually upgrade?
```
Comment (bergerhoffer) [inline release_notes/ocp-4-11-release-notes.adoc:301]: @sdodson To keep things simple, can we just add a general call to action for users to start migrating away from these APIs as of 4.11 GA? Like that they will no longer be able to use these APIs in the next release, so they should start migrating away now?

Or do we need to actually link to the KCS article with the actual steps on how to check using the alerts/APIRequestCount, and providing the way to add the admin-ack? (I wasn't sure if they could actually provide the ack early, like at 4.11.0 GA, or if would have to be done right before they upgrade to 4.12).

Comment (sdodson) [reply release_notes/ocp-4-11-release-notes.adoc:301]: That's a good point, today a call to action, review this list of APIs begin migrating away, we can leave in the details about alerts/APIRequestCount, those will be valid and will help them chase down anything that needs to be cleaned up. However we will leave out specific mention of the ACK for now.

Once we land the admin ack gate in clusters we will amend the KCS with the ack specific bits and clusters will start linking people to the KCS.

**Comment by sdodson** [comment]
LGTM, thanks.

File: release_notes/ocp-4-11-release-notes.adoc [hunk 2]
```diff
@@ -487,6 +487,15 @@ The following OpenShift CLI (`oc`) commands were removed with this release:
...
 Support for deploying custom schedulers manually has been removed with this release. Use the xref:../nodes/scheduling/secondary_scheduler/index.adoc#nodes-secondary-scheduler-about_nodes-secondary-scheduler-about[{secondary-scheduler-operator-full}] instead to deploy a custom secondary scheduler in {product-title}.
 
+[id="ocp-4-11-future-removals"]
+=== Future Kubernetes API removals
+
+The next minor release of {product-title} is expected to use Kubernetes 1.25. Currently, Kubernetes 1.25 is scheduled to remove several deprecated `v1beta1` and `v2beta1` APIs.
+
+See the link:https://kubernetes.io/docs/reference/using-api/deprecation-guide/#v1-25[Deprecated API Migration Guide] in the upstream Kubernetes documentation for the list of planned Kubernetes API removals.
+
+See link:https://access.redhat.com/articles/6955985[ Navigating Kubernetes API deprecations and removals] for information on how to check your cluster for Kubernetes APIs that are planned for removal.
```
Comment (nalhadef) [inline release_notes/ocp-4-11-release-notes.adoc:497]: It doesn't seem to make a difference in the preview, but there's an extra space at the start of the static text in the link.

**Comment by bergerhoffer** [comment]
@wangke19 Can you please review this new section for the release notes on warning about the upcoming API removals?

Preview: http://file.rdu.redhat.com/~ahoffer/2022/OSDOCS-3568/release_notes/ocp-4-11-release-notes.html#ocp-4-11-future-removals

**Pull request merged**